PINCache/PINMemoryCache.m

```diff
@@ -79,7 +79,7 @@ - (instancetype)init
         _removeAllObjectsOnMemoryWarning = YES;
         _removeAllObjectsOnEnteringBackground = YES;
 
-        #if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_4_0
+		#if TARGET_OS_IPHONE && defined(__IPHONE_OS_VERSION_MIN_REQUIRED) && !TARGET_OS_WATCH
```

Comment:
Hmm must of slipped out of the code when I was adding watch support. I have updated the PR